Profiler/CProfilerCallback.cpp

```diff
@@ -303,26 +263,6 @@ DWORD CProfilerCallback::getEventMask() {
 	return dwEventMask;
 }
 
-UINT_PTR CProfilerCallback::functionMapper(FunctionID functionId, BOOL* pbHookFunction) {
-	try {
-		// Disable hooking of functions unless in TIA mode
-#ifdef TIA
-		// FIXME this causes fail: callbackInstance != NULL && callbackInstance->config.isTiaEnabled();
-		* pbHookFunction = enableFunctionHooks;
```

Comment:
The functionmapper is only active, when COR_PRF_MONITOR_ENTERLEAVE is active. Otherwise it doesn't do anything
In a previous version, COR_PRF_MONITOR_ENTERLEAVE was set for TGA profiling but that was removed quite a while ago. Now with TIA it's called but all we do is return the original FunctionID and set pbHookFunction to true which is exactly what it does without this method as far as I understood.